src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/solicitorcreatecase/SolicitorCreateContestedAboutToSubmitHandlerTest.java

```diff
@@ -141,6 +141,37 @@ public void givenCase_whenMandatoryDataValidationFails_thenReturnsErrors() {
         Assertions.assertThat(response.getData()).isNotNull();
     }
 
+    @Test
+    public void givenCase_whenApplicantRefugeQuestionAnswered_thenApplicantRefugeTabUpdated() {
```

Comment:
It is worth taking this opportunity to upgrade the test to JUnit 5.

Change the annotation of this class to `@ExtendWith(MockitoExtension.class)` and do the refactoring.

Reply:
Thanks.  Refactored.